symfony/config/routes.yaml

```diff
@@ -1,21 +1,8 @@
-index:
-    path: /
-    controller: App\Controller\DefaultController::index
-
-# Legacy Silex Routes
-
+# 'home' route is still used in a lot of places so this redirect is left in place
+# to both continue functionality of that route without and also to redirect / to
+# the Symfony home page (/s/) when anyone goes straight to /
 home:
     path: /
-
-selectSite:
-    path: /site/select
-
-evaluation:
```

Comment:
After deploying to dev, I found one additional use of this route name that needs to be fixed. The "today's participant" view (`today.html.twig`) uses the `displayDate` macro, which dynamically generates the route name, which results in 'evaluation' for physical measurements.